README.md

```diff
@@ -144,6 +144,8 @@ If you do not have those please consider getting a [docking station](https://eu.
 
    - the device will reboot automatically.
 
+**Important:** During the installation progress, the device may reboot multiple times. The EEPROM will be automatically updated to the latest available version.
```

Comment:
I suggest to move the note about the EEPROM update to the general description at line 57
Here just mention reboots and fact that EEPROM update happens on this step